sqlmesh/core/constants.py

```diff
@@ -43,6 +43,7 @@
 MACROS = "macros"
 METRICS = "metrics"
 MODELS = "models"
+EXTERNAL_MODELS = "external_models"
```

Comment:
It was unclear if this should be `schemas` or `external_models`. Easy to change once its been decided

Reply:
@izeigerman i guess i prefer schemas/ but i'll let you make the final call here

Reply:
I strongly dislike the existing `schema.yaml` name, because "schema" as a concept is so generic and overused that it says little about the purpose of the file. Furthermore, it doesn't inform user at all about the fact that only schemas of *external* tables are expected to be there. What's even worse, "schema" means something completely different in dbt.

So far the project folder naming has been very purpose-driven, and I think we should keep it that way. If the purpose of the folder is to store definitions of external models, then we should call it as such.

Reply:
what if we also make create_external_models now create external_models.yaml by default, still accepting schema.yaml for now, but then we align and remove schema.yaml later?

Reply:
I like the idea